social.info.yml

```diff
@@ -6,11 +6,11 @@ version: '8.x-4.3'
 project: social
 
 dependencies:
-  - config
-  - system
-  - user
-  - breakpoint
-  - features
+  - drupal:config
+  - drupal:system
+  - drupal:user
+  - drupal:breakpoint
+  - features:features
```

Comment:
not `drupal:features` ?

Reply:
I think the drupal: namespace is reserved for Drupal core modules, so it should be modulename:modulename for contrib afaik. Fact is that it didnt work in https://github.com/goalgorilla/open_social/pull/754 however, so let's wait and see 🤞

Reply:
Travis tests have passed ✅

Reply:
> I think the drupal: namespace is reserved for Drupal core modules, so it should be modulename:modulename for contrib afaik. Fact is that it didnt work in #754 however, so let's wait and see 🤞

Sounds legit 👌

Reply:
As described in #1179

```
When using the project:module_name syntax for dependency declaration the
project part should reflect the name under which the project can be
found on drupal.org. For all modules contained in Open Social this is
thus the social project.
```